diff --git a/packages/client/locale/src/locales/en.ts b/packages/client/locale/src/locales/en.ts
index 70ce31598c..d7cb2edf17 100644
--- a/packages/client/locale/src/locales/en.ts
+++ b/packages/client/locale/src/locales/en.ts
@@ -7,6 +7,8 @@ export const en = {
   'close': 'Close',
   'copy': 'Copy',
   'copied': 'Copied',
+  'run': 'Run',
+  'download': 'Download',
   'copy.failed': 'Copy failed',
   'copy.value': 'Copy value',
   'copy.json': 'Copy JSON',
diff --git a/packages/client/locale/src/locales/zh.ts b/packages/client/locale/src/locales/zh.ts
index 30cee308d5..bbd95d6915 100644
--- a/packages/client/locale/src/locales/zh.ts
+++ b/packages/client/locale/src/locales/zh.ts
@@ -5,6 +5,8 @@ export const zh = {
   'close': '关闭',
   'copy': '复制',
   'copied': '复制成功',
+  'run': '运行',
+  'download': '下载',
   'copy.failed': '复制失败',
   'copy.value': '复制值',
   'copy.json': '复制 JSON',
